doc/nrf/config_and_build/bootloaders/bootloader_signature_keys.rst

```diff
@@ -1,23 +1,24 @@
-.. _ug_fw_update:
+.. _ug_fw_update_keys:
+.. _bootloader_signature_keys_guide:
 
-Firmware updates
-################
+Signature keys
+##############
 
 .. contents::
    :local:
    :depth: 2
 
-This user guide provides an overview of firmware updates with |NCS|.
+When using :ref:`bootloader` or :doc:`mcuboot:imgtool`, a signature key is used for validating the next image in the boot chain.
 
-.. _ug_fw_update_keys:
-
-Signature keys
-**************
+By default, when not explicitly defined, a private/public key pair is generated during the build when you are :ref:`ug_bootloader_adding_immutable_keys` or :ref:`ug_bootloader_adding_upgradable_mcuboot` and then used by the application.
+The following sections describe the available methods for generating the key pair.
```

Comment:
Isn't it true only for B0? IIRC MCUboot uses predefined default keys that are part of the MCUboot repository.

Reply:
We may need to globally rename `b0` to `NSIB` at some point and remove all of the `B0` occurrences in documentation/code/configuration then.

Reply:
> We may need to globally rename `B0` to `NSIB` at some point and remove all of the `B0` occurrences in documentation/code/configuration then.

Noted down in NCSIDB-964.